src/main/scala/net/ripe/rpki/publicationserver/Boot.scala

```diff
@@ -58,11 +61,18 @@ class PublicationServerApp(conf: AppConfig, https: ConnectionContext, logger: Lo
     val metrics = Metrics.get(registry)
     val metricsApi = new MetricsApi(registry)
 
-    val publicationService = new PublicationService(conf, metrics)
-
+    val dataFlusher = new DataFlusher(conf)
     // Run it asynchronously as it can be quite long, but we want
     // to start accepting HTTP(S) connections ASAP.
-    val asyncLongFSInit = Future { publicationService.initFS() }
+    val asyncLongFSInit = Future { dataFlusher.initFS() }
+
+    // Write repositories as a fixed interval
+    this.repositoryWriter = system.scheduler.scheduleAtFixedRate(
```

Comment:
this one needs to start after dataFlusher.initFS() I believe. They in principle will lock on the DB, but it looks more reasonable to me to do 
```
Future { 
   dataFlusher.initFS(); 
   this.repositoryWriter = system.scheduler.scheduleAtFixedRate(.. no delay here.. )
   }
```

Reply:
Probably not ".. no delay here.." but the delay equal to the execution period.